train_diffusion_prior.py

```diff
@@ -156,9 +214,9 @@ def train(image_embed_dim,
             if(int(time.time()-t) >= 60*save_interval):
                 t = time.time()
 
-                save_model(
+                save_diffusion_model(
                     save_path,
-                    dict(model=diffusion_prior.state_dict(), optimizer=optimizer.state_dict(), scaler=scaler.state_dict()))
+                    dict(model=diffusion_prior.state_dict(), optimizer=optimizer.state_dict(), scaler=scaler.state_dict(),hyperpmts = config))
```

Comment:
let's put  that in the function

Reply:
Fixed.

Reply:
looks like you changed the function, but the call still need to be adapted

Reply:
aargh, let me fix that

Reply:
Done.